app-editor/vim/stone.yml

```diff
@@ -0,0 +1,33 @@
+#
+# SPDX-FileCopyrightText: © 2020-2022 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : vim
+version     : 9.0.0260
+release     : 1
+homepage    : https://github.com/vim/vim
+upstreams   :
+    - https://github.com/vim/vim/archive/refs/tags/v9.0.0260.tar.gz : 6cc50bc98b7273998fbcf4e3fd4334c9a9c1a0551ec8f99e7c92f8554ad7f596
+summary     : Vi IMproved, an advanced text editor
+description : |
+    Vim, which stands for Vi IMproved, is an open-source, multiplatform text editor extended from vi. It was first released by Bram Moolenaar in 1991. Since then, numerous features have been added to Vim, many of which are helpful in editing program source code.
+license     :
+    - Vim
+builddeps   :
+    - pkgconfig(ncursesw)
```

Comment:
Are there any deps that would be good to build this with but that are not packaged yet? If so you could put some `# TODO` comments here with what they are and what they enable. Eventually as the repo expands and those are brought in we can rebuild this package with them.

Reply:
Done. They are mainly for extra providers and Vim's GTK front-end.

Reply:
Welp, I basically just copied them from the other operating system's build file.

Reply:
Hey there,

As this is a recipe monorepo with lots of other recipes, friendly reminder that the commit message needs to have version info when adding/updating packages for clarity of shortlog view.

I added this guidance to the README last night:

### Commit messages

Use a category/pkg: verb <foo> format with one commit per package change.

Example: `shells/fish: Add vX.Y` or `shells/fish: Update to vX.Y` or `shells/fish: Fix (...)`

Bump release for functional recipe changes only; do not bump release for no functional change commits (documentation typos, formatting).

No functional change commits use the [NFC] tag: `shells/fish: [NFC] Tweak comments.`

Reply:
Done.